en/developer/frameworks/articles/cache-configuration/03-caching-entity-finder-results.markdown

```diff
@@ -0,0 +1,83 @@
+---
+header-id: caching-entity-finder-results
+---
+
+# Caching Entity Finder Results
+
+[TOC levels=1-4]
+
+If your app frequently retrieves service entities based on criteria, your
+service entity finder might be good candidates to cache. In Service Builder project's
+[persistence classes](/docs/7-2/appdev/-/knowledge_base/a/understanding-the-code-generated-by-service-builder)
+Service Builder generates code that associates
+[service entity finder](/docs/7-2/appdev/-/knowledge_base/a/defining-service-entity-finder-methods)
+query parameter values with their results. There's code for caching entities,
+paginated entity lists, and non-paginaged entity lists that match your finder
+parameters. The code is bypassed by default. Configuring cache for the entity
+finder results (FinderCache) enables the generated caching code. Here you'll
```

Comment:
@jhinkey Like https://github.com/jhinkey/liferay-docs/pull/481, I think SB should enable caching unless `cache-enabled="false"` is set. Configuring a cache is for tuning (e.g. max elements in memory can sometimes affect performance).